docs/src/test/scala/docs/http/scaladsl/server/directives/FutureDirectivesExamplesSpec.scala

```diff
@@ -40,7 +40,7 @@ class FutureDirectivesExamplesSpec extends RoutingSpec {
       path("divide" / IntNumber / IntNumber) { (a, b) =>
         onComplete(divide(a, b)) {
           case Success(value) => complete(s"The result was $value")
-          case Failure(ex)    => complete((InternalServerError, s"An error occurred: ${ex.getMessage}"))
+          case Failure(ex)    => complete((InternalServerError, s"An error occurred: $ex"))
```

Comment:
So that will add the exception class in addition to the message?

Reply:
exactly